lib/Controller/EndpointController.php

```diff
@@ -247,12 +252,19 @@ protected function notificationToArray(int $notificationId, INotification $notif
 		];
 
 		if ($apiVersion !== 'v1') {
+			if ($this->request->isUserAgent([IRequest::USER_AGENT_TALK_DESKTOP])) {
+				$shouldNotify = $notification->getApp() === 'spreed';
+			} else {
+				$shouldNotify = !$hasActiveTalkDesktop || $notification->getApp() !== 'spreed';
+			}
```

Comment:
I have a 500 response here

```
Undefined constant OCP\IRequest::USER_AGENT_TALK_DESKTOP in file '/var/www/html/apps/notifications/lib/Controller/EndpointController.php' line 255
```

Does it require a server update? In this case, shouldn't we check it here for compatibility?

Reply:
git pull server yeah. It as added yesterday